README_transact.md

```diff
@@ -49,6 +49,33 @@ Dynamoid::TransactionWrite.execute do |txn|
 end
 ```
 
+### Update items
+An item can be updated by providing the hash key, range key if applicable, and the fields to update.
+Updating fields can also be done within a block using the `set()` method.
+To increment a numeric value or to add values to a set use `add()` within the block.
+Similarly a field can be removed (set to nil) or values can be removed from a set by using `delete()` in the block.
```

Comment:
`set to nil` seems incorrect now as far as `#delete` uses `REMOVE` that `Removes one or more attributes from an item.` (I suppose it meant removing key-value pair, not just assigning `nil`)

Reply:
Removed "(set to nil)".